compiler/rustc_target/src/abi/mod.rs

```diff
@@ -1070,6 +1070,12 @@ pub trait TyAndLayoutMethods<'a, C: LayoutOf<Ty = Self>>: Sized {
         cx: &C,
         variant_index: VariantIdx,
     ) -> TyAndLayout<'a, Self>;
+    fn ty_and_layout_kind(
```

Comment:
in particular, I don't think this is called from anywhere else, right?

Reply:
Yes, I have moved the helper function to now be inside the `field` method